Site layout changes

chadcurtis opened the pull request:
They're far from perfect, but here are changes made based on notes in #1. Tell me how you'd like to see this content laid out, or altered, as you see fit.

chadcurtis committed:
Initial edits to site layout.

alexgleason commented:
Good first hit at it. Check the drawing again. I'm thinking we'll want to scrap the BG image in favor of a full width banner on each page. Take some creative liberty with the design. This is way more freeform than we're used to.

MK says she'll write back with some suggestions. Good job.

marykatefain commented:
Thanks for your edits! This is mostly moving in the right direction, but I had some pretty drastic changes in mind. Here are some of the things I want to do, and some examples of sites that do those things:

1) A more complete top nav system with a place for the logo, and a banner image instead of background
[EXAMPLE 1](http://2.bp.blogspot.com/-73yVi2PHOWo/Utx9mUjeZ5I/AAAAAAAAIxk/7ic95BQdA90/s1600/Screen+Shot+2014-01-20+at+01.28.46.png)
[EXAMPLE 2](https://s3.amazonaws.com/blog.psd2html.com/wp-content/uploads/2011/12/29173526/ricebowls2.jpg)

2) Consistent theme across pages, including top nav and new banner img for each page
Pages should include: About, Mission/Values, Campaign Priorities (Legislative & Outreach), Event Calendar (Activism, Social, and Submit  Event), Contact Us, Updates, Resources

3) Events/Calendar links
[EXAMPLE](https://theeventscalendar.com/content/uploads/2014/12/example_dakota2.png)

5) Contact form

Some notes on this:
- Don't worry about graphics, just get things in place and use placeholder images. I'm not planning necessarily on sticking with the current color theme. If you can just get the layout and everything of the pages under control I can add the right pics/colors/ etc

- I was planning on completely re-doing this site over in a CMS. This version was never meant to be anything more than a basic landing page. Apostrophe would probably be good to use bc CW is trying to get in with them, but Wagtail or any other CMS you are comfortable with is fine. 

- I know you don't have much of the copy yet, so you can just use filler text to complete the "about" and events pages, etc


I know this is a lot, but any progress forward is appreciated! Thanks so much!

alexgleason commented on www/_base.njk:
```diff
@@ -7,13 +7,23 @@
+        js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.8";
+        fjs.parentNode.insertBefore(js, fjs);
+      }(document, 'script', 'facebook-jssdk'));
+    </script>
```
Any reason this script is at the top of the body tag instead of the bottom? Typically it's better to place at the bottom for improved performance. Source: [Browser Diet](https://browserdiet.com/#css-on-top-js-on-bottom)

chadcurtis committed:
Cleaned up code, added new pages.

chadcurtis committed:
Added contact page and dummy form.

chadcurtis commented:
Hey @marykatefain, sorry for the delay. I've made further edits, including:

- Altering the nav bar somewhat to fit a 'skeleton' of what you're looking for. The background color is green, but this is mostly to define it as a solid color (akin to your first example).

- I've updated the existing pages to match the same layout, while adding some of the pages you've suggested.

- Assuming the events calendar is in Google, we can embed it via [this guide](https://support.google.com/calendar/answer/41207?hl=en (a link to the calendar would help here). If it's hosted elsewhere, or if you plan to maintain the calendar on the site itself, then we may pursue some of the other options available.

- I've added a basic shell of a contact form. This may change, depending on what the CMS offers, but otherwise we can "plug it in" with a PHPMailer.

If there's no further assets/copy at this time, I can give a shot at altering the style of this site.

@alexgleason The facebook developer site suggested placing it at the top, but this seemed unconventional. I've shifted it to the bottom of the <body> tag, after hearing your advice.

alexgleason commented:
Lol, Facebook cares more that their widget loads than that your website loads...

alexgleason commented:
Just checked it out, this looks great!

alexgleason commented:
Zoomed out screenshot for reference

![screenshot from 2017-03-30 11 00 10](https://cloud.githubusercontent.com/assets/3639540/24510767/1b212bd0-1538-11e7-8c4f-d49362e38450.png)

marykatefain commented:
Awesome! This is def moving in the right direction. Thanks, Chad!

On Mar 30, 2017 11:00 AM, "Alex Gleason" <notifications@github.com> wrote:

> Zoomed out screenshot for reference
>
> [image: screenshot from 2017-03-30 11 00 10]
> <https://cloud.githubusercontent.com/assets/3639540/24510767/1b212bd0-1538-11e7-8c4f-d49362e38450.png>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/marykatefain/PhillyAnimalRightsAlliance/pull/4#issuecomment-290438586>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARH60GCEinZ_gE7sOzF3ugp_k3Hm0vH-ks5rq8OdgaJpZM4MniSK>
> .
>
